stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/viewmodel/ChannelListViewModelBinding.kt

```diff
@@ -43,4 +45,18 @@ public fun ChannelListViewModel.bindView(
     view.setOnEndReachedListener {
         onAction(ChannelListViewModel.Action.ReachedEndOfList)
     }
+
+    view.setChannelDeleteClickListener {
+        AlertDialog.Builder(view.context)
```

Comment:
The sample app now shows this new AlertDialog by default, because it calls `bindView` after setting up a bunch of listeners. We should fix that `bindView` call and make sure it happens before other customizations, and it'd be good to check the other `bindView` usages in the sample app as well.

Reply:
Good catch 😓 Fixed.